substrate/primitives/runtime/src/lib.rs

```diff
@@ -954,6 +954,23 @@ pub fn print(print: impl traits::Printable) {
 	print.print();
 }
 
+/// Utility function to declare string literals backed by an array of length N.
+/// The input can be shorter than N, in that case the end of the array is padded with zeros.
```

Comment:
```suggestion
///
/// The input can be shorter than N, in that case the end of the array is padded with zeros.
```
Nit to stick with the docs guidelines